cockatrice/src/tab_game.cpp

```diff
@@ -100,6 +100,8 @@ DeckViewContainer::DeckViewContainer(int _playerId, TabGame *parent)
 {
     loadLocalButton = new QPushButton;
     loadRemoteButton = new QPushButton;
+    if (parentGame->getIsLocalGame())
```

Comment:
I was following the same naming scheme as in the `TabSupervisor` class to avoid confusion. We should probably keep the naming scheme as consistent as possible.